code-studio/src/js/initApp/initApp.js

```diff
@@ -93,6 +95,23 @@ window.apps = {
         var lastServerResponse = reporting.getLastServerResponse();
         if (lastServerResponse.videoInfo) {
           showVideoDialog(lastServerResponse.videoInfo);
+        } else if (lastServerResponse.endOfStageExperience) {
+          let body = document.createElement('div');
+          let stageInfo = lastServerResponse.previousStageInfo;
+          let stageName = `${window.dashboard.i18n.t('stage')} ${stageInfo.position}: ${stageInfo.name}`;
```

Comment:
the three lines above can all be const rather than let, yes?

const in JS is a somewhat different concept than in other languages. In general, if something can be const (i.e. you're not modifying that variable in the future) it should be. This can be somewhat counterintuitive at times, as something like the following is entirely valid:
```
const foo = { x: 1, y: 2};
foo.y = 3;
```
(the const-ness only applies to the top level variable)